core/Plugin/Menu.php

```diff
@@ -30,16 +30,6 @@
  */
 class Menu
 {
-    protected $module = '';
```

Comment:
Given it is used only in the URL generator methods I think it's safe to think that there will be no performance regression?